charts/jaeger/templates/tracing-rbac.yaml

```diff
@@ -0,0 +1,26 @@
+---
+###
+### linkerd-collector RBAC
+###
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: linkerd-collector
```

Comment:
Since this will live in the `linkerd-jaeger` namespace, I think we can simply call this `collector`